tests/gnrc_tcp/Makefile

```
@@ -0,0 +1,47 @@
+include ../Makefile.tests_common
+
+# Basic Configuration
+BOARD ?= native
+PORT ?= tap0
```

```suggestion
TAP ?= tap0
```

`PORT` is the tty-PORT

---

I think the PORT semantics are somewhat convoluted.
Under native: PORT continas the tap device to use.
Under non-native: PORT is tty.

I'll think of something

---

> I think the PORT semantics are somewhat convoluted.
> Under native: PORT continas the tap device to use.
> Under non-native: PORT is tty.
> 
> I'll think of something

Because `native` does not require a serial port it was (yes somewhat mis-) used to provide the `tap0` interface (which on the other hands real boards did not require at least back then). Other than that the semantics of PORT are very clear: The serial port.